tests/transactions/builder/test_htlc_lock.py

```diff
@@ -103,8 +109,20 @@ def test_htlc_lock_transaction_custom_fee():
     assert transaction_dict['typeGroup'] == 1
     assert transaction_dict['typeGroup'] == TRANSACTION_TYPE_GROUP.CORE.value
     assert transaction_dict['fee'] == 5
-    assert transaction_dict['asset']['lock']['secretHash'] == '0f128d401958b1b30ad0d10406f47f9489321017b4614e6cb993fc63913c5454'
+    assert transaction_dict['asset']['lock']['secretHash'] == secret_hash
     assert transaction_dict['asset']['lock']['expiration']['type'] == 1
     assert transaction_dict['asset']['lock']['expiration']['value'] == 1573455822
 
     transaction.schnorr_verify()  # if no exception is raised, it means the transaction is valid
+
+
+def _generate_secret_hash():
+  secret = "super secret code that must be unique and entirely random"
+
+  # secret code is a key ingredient used to unlock the htlc lock. To generate an unlock code one
+  # must "hexify" it: `unlock_secret = hexlify(secret_h.encode()).decode()``
```

Comment:
having this explanation somewhere in a test would have saved me a lot of time. Trying to figure out why my transaction was being rejected and how to properly construct an `unlock_secret` was annoying.